pkgs/applications/graphics/rawtherapee/default.nix

```diff
@@ -3,13 +3,13 @@
 }:
 
 stdenv.mkDerivation rec {
-  version = "5.0-r1";
+  version = "5.0-r1-gtk2";
```

Comment:
I'm not sure if that's a good idea. `builtins.versionCompare "5.0-r1-gtk2" "5.0-r1"` says the `gtk2` version is greater. This will break `nix-env -u` when the suffix is removed at some point.